test/src/main/configurations/JDBC/ConfigurationManageDatabaseWrapper.xml

```diff
@@ -2,7 +2,7 @@
 	<adapter name="ManageDatabaseWrapper"
 		description="Wrapper for ManageDatabase to retry until a specific number of results is returned to dynamically solve timing issues"
 		active="${manageDatabase.active}">
-		<receiver name="ManageDatabaseWrapper">
+		<receiver name="ManageDatabaseWrapper" transactionAttribute="NotSupported">
```

Comment:
❗ ❓ 🤯 

Deze vind ik een beetje twijfelachtig eerlijkgezegd?

Reply:
Ja ik ben nog aan het knoeien om de laatste BTM test groen te krijgen.. Het is alleen Oracle die problemen heeft. In de log zie ik dat iets naar de ErrorStorage verplaatst wordt, maar blijkbaar ziet dit ding dat niet.